src/pkgmt/cli.py

```diff
@@ -95,11 +95,32 @@ def hook(uninstall, run):
     default=False,
     help="Do not ask for confirmation",
 )
-@click.option("--push/--no-push", default=True)
-@click.option("--tag/--no-tag", default=True)
+@click.option("--push/--no-push", default=None)
+@click.option("--tag/--no-tag", default=None)
 @click.option("--target", default=None)
 def version(yes, push, tag, target):
     """Create a new package version"""
+
+    cfg_tag, cfg_push = config.PyprojectConfig().get_version_configurations()
+
+    if tag is not None:
+        if cfg_tag is not None and tag != cfg_tag:
+            print(
+                f"Value of 'tag' from CLI : {tag}. "
+                f"This will override tag={cfg_tag} as configured in pyproject.toml"
+            )
+    else:
+        tag = cfg_tag if cfg_tag is not None else True
+
+    if push is not None:
+        if cfg_push is not None and push != cfg_push:
+            print(
+                f"Value of 'push' from CLI : {push}. "
```

Comment:
do not add whitespace before the colon

```suggestion
                f"Value of 'push' from CLI: {push}. "
```

Reply:
Removed whitespace